### PR TITLE
Add credits information

### PR DESCRIPTION
**Pull request by choldgraf**
Adds some information about how to add and use credits on AWS and Google. I'm not sure how to do credits on Azure yet so we can add that when we know what to do.

closes https://github.com/2i2c-org/infrastructure/issues/1142

### CODE REVIEW

File: docs/topic/credits.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+This page has information about how this works for each major cloud provider we use.
+
+## Amazon Web Services
+
+On Amazon Web Services, credits come via **Promotion Codes**.
+These are unique strings that are attached to a given credit offer.
+We can apply these codes to any Account in AWS.
+To do so, follow these steps:
+
+1. Log in to the relevant account via [our AWS SSO](cloud-access:aws-sso).
```
Comment (yuvipanda) [inline docs/topic/credits.md:13]: Add a note that credits only apply from the *next billing period*, so next calendar month?

File: docs/topic/credits.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+2. Confirm that you are in the right account! Credits will *only be available for this account*, and unavailable in any other accounts in the 2i2c AWS organization.
+3. Go to {guilabel}`AWS Billing` -> {guilabel}`Credits`
+4. Click {guilabel}`Redeem Credit` in the top-right.
+5. Enter the Promotion Code and click {guilabel}`Redeem Credit` in the bottom-right.
+
+### Expiration dates
```
Comment (choldgraf) [inline docs/topic/credits.md:19]:
```suggestion
5. Enter the Promotion Code and click {guilabel}`Redeem Credit` in the bottom-right.

:::{warning}
Credits will be applied to the **next billing period** (generally the next calendar month).
If you apply credits in the middle of the month, they will not cover any of that month's usage.
:::

### Expiration dates
```

Comment (choldgraf) [reply docs/topic/credits.md:19]: how's that @yuvipanda ?

Comment (yuvipanda) [reply docs/topic/credits.md:19]: @choldgraf sounds good! Can you also add the aws docs link the support person sent us?

**Review by yuvipanda — APPROVED** [review]
Good to go after adding the suggested link

File: docs/topic/credits.md (hunk 2 above)
Comment (choldgraf) [reply docs/topic/credits.md:19]: how's that?

Comment (yuvipanda) [reply docs/topic/credits.md:19]: @choldgraf looks great! Merge whenever you feel like

File: docs/topic/cloud-access.md [hunk 3]
```diff
@@ -152,4 +152,7 @@ are used to provide access to the AWS account from your terminal.
 
    When you want to use these credentials, you can simply run `export AWS_PROFILE=<your-cluster-name>`.
    This helps manage multiple sets of credentials easily. You can validate this works by running
-   `aws sts get-caller-identity`.
\ No newline at end of file
+   `aws sts get-caller-identity`.
+
+
```
Comment (choldgraf) [inline docs/topic/cloud-access.md:157]:
```suggestion
```

**Pull request merged**